src/PowerShellEditorServices/Services/PowerShell/Handlers/GetCommentHelpHandler.cs

```diff
@@ -77,8 +68,7 @@ public async Task<CommentHelpRequestResult> Handle(CommentHelpRequestParams requ
 
             List<string> helpLines = ScriptFile.GetLinesInternal(helpText);
 
-            if (helpLocation != null &&
-                !helpLocation.Equals("before", StringComparison.OrdinalIgnoreCase))
+            if (helpLocation?.Equals("before", StringComparison.OrdinalIgnoreCase) == false)
```

Comment:
```suggestion
            if (helpLocation is not "before")
```

re: consider `is`. Also nice to skip the `== false` nullable stuff.